bucket/mednafen.json

```diff
@@ -0,0 +1,33 @@
+{
+    "version": "1.24.1",
+    "homepage": "https://mednafen.github.io/",
+    "description": "Emulator for different systems",
+    "license": "GNU GPLv2",
+    "architecture": {
+        "64bit": {
+            "url": "https://mednafen.github.io/releases/files/mednafen-1.24.1-win64.zip",
+            "hash": "ad3a7906521f0ce45737c2f0f7ea1117799548e60d10a746ac74ddd26d944cbb"
+        },
+        "32bit": {
+            "url": "https://mednafen.github.io/releases/files/mednafen-1.24.1-win32.zip",
+            "hash": "d26456eb711feac679e424968edee579b69c38ce1972a93bed10355b04ff21cd"
+        }
+    },
+    "bin": "mednafen.exe",
+    "shortcuts": [
+        [
+            "mednafen.exe",
+            "Mednafen"
+        ]
+    ],
```

Comment:
```suggestion
    "pre_install": "if (-not (Test-Path \"$persist_dir\\mednafen.cfg\")) { New-Item \"$dir\\mednafen.cfg\" | Out-Null }",
    "bin": "mednafen.exe",
```

Reply:
Thanks a lot for this! I was hesitant on mentioning that I didn't know how to create a persist for the configuration files that aren't within directories.